environments/staging/readme.md

```diff
@@ -9,11 +9,53 @@
 - APM: `0x4994a4b1215bf73f0e69ce69c2e6b43c496ad4bf`
 - DAOFactory (Aragon 0.6): `0x10e1fcca61798cae1e1bcddcbc0cf3e8c03418a6`
 - DAOFactory (Aragon 0.7): `0xbca2c99aa7018edcde60e6c5744f68003112535d`
+- DAOFactory (Aragon 0.8): `0xf959e8953e2fe03782a7b86a7a5d948cb511ef5d`
 
 ## Deployments
 
 -----------
 
+- Version: Aragon client 0.8
+- Start date: September 3rd 2019, 18.30pm CEST
+- Finish date: TBD
+- Deployer: ([@facuspagnuolo](https://github.com/facuspagnuolo)) [`0x03acbcb547d03c8e7746ef5988012b59604aa083`](https://rinkeby.etherscan.io/address/0x03acbcb547d03c8e7746ef5988012b59604aa083)
+
```

Comment:
Can we add the aOS commit hash here as we usually do for these?

https://github.com/aragon/deployments/blob/a9fc9ba0068f249aea6d686f224762d29edf5ccf/environments/staging/readme.md#aragonos-1